pa/biGraph.ml

```diff
@@ -0,0 +1,176 @@
+open Pervasives
```

Comment:
"matching" might be a more appropriate name than "bigraph" for this file (following the convention that <module>.Make is a functor that makes a <module> -- BiGraph.Make makes an implementation of the matching algorithm).